templates/OperationOptionsSnippet.stencil

```diff
@@ -1,14 +1,17 @@
 // Process endpoint options
 if let options = options {
     // Query options
-    {% for opt in op.options.queryOptions %}
-    if let {{ opt.name }} = options.{{ opt.name }} { queryParams.append("{{ opt.name }}", {{ opt.name}}) }
+    {% for param in op.optionalQueryParams %}
```

Comment:
So with my above suggestion, this would look like `for param in op.params.query.optional`